client/app/cloud/project/billing/cloud-project-billing-agora.service.js

```diff
@@ -0,0 +1,44 @@
+angular.module('managerApp')
+  .service('CloudProjectBillingAgoraService',
+    class {
+      /* @ngInject */
+      constructor(
+        OvhApiCloudProjectServiceInfos,
+        OvhApiMeConsumption,
+        OvhApiServiceRenewForecast,
+      ) {
+        this.OvhApiCloudProjectServiceInfos = OvhApiCloudProjectServiceInfos;
+        this.OvhApiMeConsumption = OvhApiMeConsumption;
+        this.OvhApiServiceRenewForecast = OvhApiServiceRenewForecast;
+      }
+
+      getProjectServiceInfos(serviceName) {
+        return this.OvhApiCloudProjectServiceInfos.v6().get({ serviceName }).$promise;
+      }
+
+      getCurrentForecast(serviceId) {
+        return this.OvhApiMeConsumption.Usage().Forecast().v6().get().$promise
+          .then(forecast => _.find(forecast, { serviceId }));
+      }
+
+      getCurrentConsumption(serviceId) {
+        return this.OvhApiMeConsumption.Usage().Current().v6().get().$promise
+          .then(consumption => _.find(consumption, { serviceId }));
+      }
+
+      getBillForecast(serviceId) {
+        return this.OvhApiServiceRenewForecast.v6()
+          .get({ serviceId, includeOptions: true }).$promise;
+      }
+
+      static formatPrice(value, currencyCode) {
```

Comment:
This helps to format legacy data to the new format (to avoid duplicate the view) and allows to keep the same format when some operation (e.g : adding) is necessary